mne/io/base.py

```diff
@@ -2409,10 +2410,15 @@ def concatenate_raws(raws, preload=None, events_list=None, verbose=None):
     Parameters
     ----------
     raws : list
-        List of Raw instances to concatenate (in order).
+        List of `~mne.io.Raw` instances to concatenate (in order).
     %(preload_concatenate)s
     events_list : None | list
-        The events to concatenate. Defaults to None.
+        The events to concatenate. Defaults to ``None``.
+    on_mismatch : 'raise' | 'warn' | 'ignore'
+        How to handle situations when we discover that specific aspects of the
+        `~mne.io.Raw` instances mismatch in a way that could cause problems
+        after concatenation. Currently, this check is limited to the
+        device-to-head transformation. Defaults to ``'raise'``.
```

Comment:
What you say here about `device-to-head`-only is how it should work, but this is not how it's coded